### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BhargaviChada**
Added link to guides in the learn section.

##### Description of change

Added link to guides in learn section.

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [ x] Documentation is changed or added
- [ ] Lint process passed (`npm run lint`)
- [ ] Tests passed (`npm run test`)

### CODE REVIEW

**Review by ivanpopelyshev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ PixiJS has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamles
 - Getting started: Check out @kittykatattack's comprehensive [tutorial](https://github.com/kittykatattack/learningPixi).
 - Examples: Get stuck right in and play around with PixiJS code and features right [here](http://pixijs.github.io/examples/)!
 - Docs: Get to know the PixiJS API by checking out the [docs](https://pixijs.github.io/docs/).
+- Guide: Supplementary resource to the API documentation [here](https://github.com/pixijs/guides/tree/main/docs).
```
Comment (bigtimebuddy) [inline README.md:40]: Link should be https://pixijs.io/guides

**Review by bigtimebuddy — CHANGES_REQUESTED** [review]

**Comment by domis86** [comment]
when merged it should close task #7617

**Comment by BhargaviChada** [comment]
> when merged it should close task #7617

Sorry forgot to mention 'Closes #7616' in the previous commit

**Review by bigtimebuddy — APPROVED** [review]
Thank you!

**Pull request merged**